who_are_we.md

```diff
@@ -0,0 +1,63 @@
+
+The Gizra story is about a way, an outlook, a philosophy that navigates all the internal and external aspects of a web development company. It is knowing and acting on the understanding that *we can do better.*
+
+It is summed up nicely under four sections of The Gizra Way. Let’s look at them:
+
+**Transparency**.  Everything is out in the open, warts and all.  This may be new to you and may take some adjustment. We set the tone as early as our price estimations by basing them on a detailed breakdown of project tasks (small chunks of 12 hours or less) with an attached cost.  This extends into our projects where our clients have full access to Github. They see everything - including our internal communications. Even this book is public. 
+
+**Open Communication**. We work hard to create a common language with our clients and to set expectations early on and this applies internally as well.  
+
+**Meticulous Execution**. This area is the core of how well we do things.  We implement tested processes to ensure the highest technical standards. For example, a complete deployment system is built into our projects, we use our own Gizra developed testing toolkit, and do pixel-perfect design implementation.
+
+**Investment in you and the community**. We can do better is about bettering ourselves and others as an organization and also as individuals.  You can become better by embracing our mentoring process that will usually keep you on your toes - being a little uncomfortable means you are stretching your bounds.  We also like seeing you want to contribute and encouraged you to do so to the open-source community on company time.
+
+Keep in mind and check yourself against these Gizra values. They will guide you in situations where you still don’t have an answer and set the standards you should want to reach.
+
+###Our Purpose:
+
+A general problem our industry faces is the high failure rate of web projects due to bad project management. The projects usually do not meet any one of the three criteria of success: being on time, in budget,happy users. 
+
+So, if we intersect the Gizra Way with what is going on outside, our purpose becomes clear:
+
+**For Gizra, the world will be a better place if more web projects realize that there is a better way.**
+
+###Our Position
+
+Our purpose probably strikes a chord with anyone who has implementing an IT project of any kind.  We want to reach the clients that can appreciate that there needs to be a better way.  Hence, we believe our message best reaches seasoned executive who have seen enough web projects to get burned. They may have already resigned to the idea that that is how projects are run but know deep down that there has to be another way. Our target clients are most probably in medium sized organizations but really it's about any place a brave executive can successfully influence their organization to adopt a new, simple, and sometimes radical, way of doing things.
+
+**For** medium sized companies
```

Comment:
> What we have done until now is not necessarily what should or can be done from now on.

I think this part encapsulates the problem I see with the current PR - it's a bit of mixture of:
1. Out narrative
1. Marketing agenda
1. Future assumptions.

I think we should focus on this chapter only on the narrative. Narrative not necessarily just as an old story, but how we currently or up until now worked.

It might sounds a bit abstract, I don't have an exact vision of how it should look, but I think about the new dev/ account manager coming and trying to understand where did they get to. This is the audience we should focus on. They should know for example:

* How being friendly isn't just a nice to have, but it's part of being professional
* We are friendly, but not "family"
* How we iterate over our success and failues to learn form there
* How we believe in transparency
* Our Thursday toasts
* Why we believe devs should sit around the same table
* How we organize the milk in the frig (a.k.a. Roy's system). It's a bit of a joke obviously, but it shows how the Gizra way is influencing us